randovania/games/cave_story/exporter/options.py

```diff
@@ -9,13 +9,15 @@
 
 @dataclasses.dataclass(frozen=True)
 class CSPerGameOptions(PerGameOptions):
-    output_directory: Path | None = None
+    output_directory: Path | None = (None,)
+    platform: str | None = None
```

Comment:
what is the point of this being `str | None`?

Reply:
I assumed str fields being nullable was standard practice in the exporter classes. I noticed that's how it was handled for Dread's exporter (which I based my implementation off of), so I just followed how it was done there.

Reply:
should probably be an enum anyway

Reply:
Changed it to use the `CSPlatform` enum